### ISSUE TITLE
State of Notifiers

### ISSUE THREAD

**Issue by liiight** [notifiers/notifiers#384]
tl;dr: This project **IS NOT NEGLECTED** and new changes will happen in the near future.

I opened this issue a way to address the people who use and rely on this library (as the gitter channel is mostly abandoned).
COVID-19 has hit the world hard these last few months, and while I am very lucky to have only my availability affected by it, it has been greatly affected nonetheless. As I address my commitments to my family and my job, open source projects I work on obviously are not the highest priority right now.

That being said, this project is very important to me and it is most definitely NOT ABANDONED. As the effects on the pandemic seem to (hopefully) wind down, regular life will resume its course, and working on notifiers will resume as well. I have greate plans for this project in the future and I'm anxious to push them out into the world. 

I'd like to thank everyone who enjoys this library (that has recently surpasses 1M total downloads!) for making my little idea come to fruition in such a popular manner.

Or Carmi

**Comment by liiight** [comment]
Well, it's been about a year since that original post and unfortunately I still haven't gotten around to put some time on this project.

I want to sincerely apologize for all the users of this library for not being able to properly maintain it. This project is very dear to me and there isn't a day that goes by when I do not give it some thought (and feel guilt for not contributing to it).

Hopefully my life situation will become more sustainable in the neat future and I'll have the time/energy/motivation to put some work into it.

The next major release is almost ready, piling dust on a feature branch. After I'll be able to release it, ill make an active effort to get more maintainers to this project so I won't become it's bottle neck.

I hope you're all doing well at these difficult times and I promise this project will not fall into the obscurity of oblivion.

- Or

**Comment by liiight** [comment]
I'm happy to write that I have resumed working on notifiers. My plan is to get the project to a very stable state in terms of CI/CD, release a new major version that will hopefully get as little to no breaking issues, and then try to add maintainers to it, so I won't become its bottle neck anymore

**Comment by jorenham** [comment]
Can we get an update?

**Comment by liiight** [comment]
I can't believe it's been 4 years since I made that post.

At this point, I have to face the music, I don't have the bandwidth to support this project by myself, a statement that truly saddens me to make. The passion and connection I have for this project is in direct conflict with how much I abandoned it, and it is truly a shitty feeling.

But I'm not ready to throw in the towel, not yet. 
I have learned SO MUCH since I started this project, and the ecosystem has shifted so drastically that I feel I simply can not leave it as is.

So my plan is to force myself to grind out and make a version 2.0 which would basically be a rewrite, which will improve on the core structure, use a few more deps (like attrs and pydantic) and have the project in a state that would make it a lot more easier to contribute to and scale from.

I already started going this route, and I plan to push through and make this happen in an actually reasonable time span (for real this time).

I'd like to thank everyone who showed and still shows interest in the project. It's truly a good feeling to know that others find this useful. So thank you 🙏

**Comment by liiight** [comment]
Just released v1.3.5 to pypi, with minor stuff that was added and migration to uv.
I got some free time on my hands and I want a (sort of) clean slate before I start pushing for notifiers 2.0.

**Comment by liiight** [comment]
I'm trying to revive this project and I want to keep my momentum going.
I'd love to get more contributors for this project. 

If you want to help, I'm looking for help in the maintenance sections, specifically workflows, documentation, publishing automations and etc. Anything that will help setup the infra to be more robust and sustainable would be welcome and will allow me to push towards v2.0.
